src/app/appform/empty.tpl.html

```diff
@@ -0,0 +1,4 @@
+<div class="empty-state" layout-padding layout-margin>
+  <h2 class="md-title">No form associated.</h2>
+  <p class="md-body-1">If you use here forms from Red Hat Mobile Application Platform, go there and associate this project with one or more forms.</p>  
```

Comment:
Something like: "If you want to use the Drag & Drop Apps Forms feature of the Red Hat Mobile Application Platform, associate this project with one or more forms" maybe?